lighthouse-core/third_party/promise-polyfill/promise.js

```diff
@@ -0,0 +1,257 @@
+/*
```

Comment:
@rviscomi for another `third_party/` thing to delete for import. Sorry about this, but we need to modify it to be able to use.

Good news is that this is for testing only, so you can just delete? Not sure how you deal with `third_party/traceviewer-js/` today